notifications/api.py

```diff
@@ -184,3 +189,25 @@ def send_comment_notifications(post_id, comment_id, new_comment_id):
 
         notifier = comments.CommentNotifier(notification_settings)
         notifier.create_comment_event(subscription, new_comment_id)
+
+
+def send_moderator_notifications(post_id, channel_name):
+    """
+    Sends post notifications to channel moderators
+
+    Args:
+        post_id (str): base36 post id
+        channel_name (str): channel_name
+    """
+
+    channel_api = get_admin_api()
+    for moderator in channel_api.list_moderators(channel_name):
+        self_user = User.objects.get(username=moderator.name)
+        notification_setting, _ = NotificationSettings.objects.get_or_create(
+            user=self_user,
+            trigger_frequency=FREQUENCY_IMMEDIATE,
+            notification_type=NOTIFICATION_TYPE_MODERATOR,
+        )
```

Comment:
This record should already exist since it should show up in the `/settings/notifications`, but we don't need to necessarily deal with that UI in this PR (should open an issue for it though, I assume non-moderator users shouldn't see it).

So, it should be added here https://github.com/mitodl/open-discussions/blob/be26dcf13f08fdf1998a801a8ed71e1fa629ab86/notifications/api.py#L28

Additionally, when this gets released, the management command to backpopulate these records should be run: 

https://github.com/mitodl/open-discussions/blob/master/notifications/management/commands/populate_notification_settings.py

Reply:
I added an issue for the /settings/notifications update
https://github.com/mitodl/open-discussions/issues/3244